app/src/app/components/specmath-modal/modal.component.html

```diff
@@ -36,37 +36,45 @@ <h3>
     </div>
     <mat-horizontal-stepper id="modal-stepper" linear #stepper>
       <mat-step>
-        <app-spec-name-input (options)="handleSpecNameInputOptions($event)"></app-spec-name-input>
+        <app-spec-name-input
+          (options)="handleSpecNameInputOptions($event)"
+        ></app-spec-name-input>
       </mat-step>
       <mat-step [optional]="true">
-        <app-defaults-file-upload (options)="handleDefaultsFileUploadOptions($event)"></app-defaults-file-upload>
+        <app-defaults-file-upload
+          (options)="handleDefaultsFileUploadOptions($event)"
+        ></app-defaults-file-upload>
+      </mat-step>
+      <mat-step>
+        <app-spec-files-upload
+          (options)="handleSpecFilesUploadOptions($event)"
+        ></app-spec-files-upload>
       </mat-step>
-      <mat-step> </mat-step>
       <mat-step> </mat-step>
     </mat-horizontal-stepper>
     <div class="modal-step-controls-container">
       <button
         id="modal-button-back"
         mat-button
         class="modal-step-button-secondary"
-        aria-label="Back button for step 2"
+        aria-label="Back button for stepper"
```

Comment:
Does "stepper" make sense from the user's perspective? Maybe "Back button to go back to the previous step"?

Reply:
Updated